### ISSUE TITLE
Exception in PHPStorm 98.145 after running tests

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#149]
```
What steps will reproduce the problem?
1. Run IDE
2. Start server
3. Attach browser
4. Run tests

After all test are finished IDE throws the following exception:


java.net.SocketException: Connection reset: java.net.SocketException: 
Connection reset
java.lang.RuntimeException: java.net.SocketException: Connection reset
    at com.google.jstestdriver.idea.RemoteTestListener$1.finished(Unknown Source)
    at com.intellij.util.concurrency.SwingWorker$1.run(SwingWorker.java:148)
    at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:319)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:597)
    at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:627)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:508)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:405)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:369)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:269)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:184)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:174)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:169)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:161)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
Caused by: java.net.SocketException: Connection reset
    at java.net.SocketInputStream.read(SocketInputStream.java:168)
    at java.net.SocketInputStream.read(SocketInputStream.java:182)
    at java.io.ObjectInputStream$PeekInputStream.peek(ObjectInputStream.java:2249)
    at java.io.ObjectInputStream$BlockDataInputStream.peek(ObjectInputStream.java:2542)
    at java.io.ObjectInputStream$BlockDataInputStream.peekByte(ObjectInputStream.java:2552)
    at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1297)
    at java.io.ObjectInputStream.readObject(ObjectInputStream.java:351)
    ... 15 more



java.net.SocketException: Connection reset
    at java.net.SocketInputStream.read(SocketInputStream.java:168)
    at java.net.SocketInputStream.read(SocketInputStream.java:182)
    at java.io.ObjectInputStream$PeekInputStream.peek(ObjectInputStream.java:2249)
    at java.io.ObjectInputStream$BlockDataInputStream.peek(ObjectInputStream.java:2542)
    at java.io.ObjectInputStream$BlockDataInputStream.peekByte(ObjectInputStream.java:2552)
    at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1297)
    at java.io.ObjectInputStream.readObject(ObjectInputStream.java:351)
    at com.google.jstestdriver.idea.RemoteTestListener$1.finished(Unknown Source)
    at com.intellij.util.concurrency.SwingWorker$1.run(SwingWorker.java:148)
    at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:319)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:597)
    at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:627)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:508)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:405)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:369)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:269)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:184)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:174)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:169)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:161)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)




```

Original issue reported on code.google.com by `olegelif...@gmail.com` on 15 Oct 2010 at 6:43

**Comment by GoogleCodeExporter** [comment]
Original comment by `aeagle22206` on 15 Oct 2010 at 3:38

**Comment by GoogleCodeExporter** [comment]
```
I'm seeing a similar exception:


java.io.UTFDataFormatException: java.io.UTFDataFormatException
java.lang.RuntimeException: java.io.UTFDataFormatException
    at com.google.jstestdriver.idea.RemoteTestListener$1.finished(Unknown Source)
    at com.intellij.util.concurrency.SwingWorker$1.run(SwingWorker.java:148)
    at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:319)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:633)
    at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:655)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:509)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:408)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:372)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
Caused by: java.io.UTFDataFormatException
    at java.io.ObjectInputStream$BlockDataInputStream.readUTFSpan(ObjectInputStream.java:3082)
    at java.io.ObjectInputStream$BlockDataInputStream.readUTFBody(ObjectInputStream.java:3007)
    at java.io.ObjectInputStream$BlockDataInputStream.readUTF(ObjectInputStream.java:2820)
    at java.io.ObjectInputStream.readUTF(ObjectInputStream.java:1051)
    at java.io.ObjectStreamClass.readNonProxy(ObjectStreamClass.java:649)
    at java.io.ObjectInputStream.readClassDescriptor(ObjectInputStream.java:809)
    at java.io.ObjectInputStream.readNonProxyDesc(ObjectInputStream.java:1565)
    at java.io.ObjectInputStream.readClassDesc(ObjectInputStream.java:1496)
    at java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:1732)
    at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1329)
    at java.io.ObjectInputStream.readObject(ObjectInputStream.java:351)
    ... 15 more



java.io.UTFDataFormatException
    at java.io.ObjectInputStream$BlockDataInputStream.readUTFSpan(ObjectInputStream.java:3082)
    at java.io.ObjectInputStream$BlockDataInputStream.readUTFBody(ObjectInputStream.java:3007)
    at java.io.ObjectInputStream$BlockDataInputStream.readUTF(ObjectInputStream.java:2820)
    at java.io.ObjectInputStream.readUTF(ObjectInputStream.java:1051)
    at java.io.ObjectStreamClass.readNonProxy(ObjectStreamClass.java:649)
    at java.io.ObjectInputStream.readClassDescriptor(ObjectInputStream.java:809)
    at java.io.ObjectInputStream.readNonProxyDesc(ObjectInputStream.java:1565)
    at java.io.ObjectInputStream.readClassDesc(ObjectInputStream.java:1496)
    at java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:1732)
    at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1329)
    at java.io.ObjectInputStream.readObject(ObjectInputStream.java:351)
    at com.google.jstestdriver.idea.RemoteTestListener$1.finished(Unknown Source)
    at com.intellij.util.concurrency.SwingWorker$1.run(SwingWorker.java:148)
    at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:319)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:633)
    at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:655)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:509)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:408)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:372)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)

```

Original comment by `aeagle22206` on 21 Nov 2010 at 7:10
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `aeagle22206` on 11 Apr 2011 at 7:33
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
```
This bug was fixed in the plugin v1.3 or higher (IDEA v10.0 or higher is 
required).
@Oleg: Could you please verify?
```

Original comment by `Sergey.S...@gmail.com` on 10 May 2011 at 7:53
- Changed state: **Fixed**
